components/dash-html-components/scripts/data/attributes.json

```diff
@@ -1080,4 +1080,4 @@
             "value"
         ]
     }
-}
+}
```

Comment:
Need to add a newline at the end, editor with auto newline at the end will conflict every time it's open.